ecs_deploy/cli.py

```diff
@@ -517,8 +521,11 @@ def record_deployment(revision, api_key, app_id, region, comment, user):
     api_key = getenv('NEW_RELIC_API_KEY', api_key)
     app_id = getenv('NEW_RELIC_APP_ID', app_id)
     region = getenv('NEW_RELIC_REGION', region)
-
+    revision = getenv('NEW_RELIC_REVISION', revision)
+    
     if not revision or not api_key or not app_id:
+        click.secho('Missing required parameters for recording New Relic deployment.' \
```

Comment:
I would show the warning only, if `api_key` is provided, because i think most users are not using New Relic (just guessing here) and they then would always see this message, even if they do not intend to use New Relic.